ports/biolab/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "biolab.zip",
+  "items": [
+    "Biolab Disaster.sh",
+    "biolab"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Biolab Disaster",
+    "porter": [
+      "scemino"
+    ],
+    "desc": "A jump'n'gun platformer, displaying many of high_impacts capabilities.",
+    "desc_md": null,
+    "inst": "Ready to run.",
+    "inst_md": null,
+    "genres": [
+      "platformer"
+    ],
+    "image": null,
+    "rtr": true,
+    "exp": true,
```

Comment:
This line, if its experimental ti won'T show up in Portmaster unless you know what you're doing.